src/reducers/tweets.js

```diff
@@ -0,0 +1,13 @@
+import { TWEET_RECEIVED } from '../actions';
+
+export default function (state = [], action) {
+  switch (action.type) {
+  case TWEET_RECEIVED:
+    return [
+      ...state.slice(-99),
+      action.payload
+    ];
+  default:
+    return state;
+  }
+}
```

Comment:
Apropos vår tidligere diskusjon. Når jeg leser denne kodesnutten må jeg tilbake til action creatoren for å skjønne hva som foregår. Hadde man brukt `action.tweet` i stedet for `action.payload` så hadde ting vært mer leselig. Leser FSA nå, hva med `action.payload.tweet` ?